code/data_structures/src/stack/infix_to_postfix/infix_to_postfix_1.cpp

```diff
@@ -0,0 +1,130 @@
+#include <iostream>
+#include <stack>
+
+class InfixToPostfix
+{
+
+public:
+    void getInfixExpression();
+
+    int getPrecedenceOfOperators(char);
+
+    void convertInfixToPostfix();
+
+private:
+    std::string expression_;
+
+};
+
+void InfixToPostfix::getInfixExpression()
+{
+    std::cout << "\nEnter Infix Expression  : ";
```

Comment:
This shouldn't be part of the class. Instead, make a constructor which takes the infix expression as input.